styles/blacklight-overrides.css

```diff
@@ -0,0 +1,33 @@
+.applied-filter .btn-outline-secondary {
```

Comment:
We already have https://github.com/sul-dlss/component-library/blob/main/selected_item/index.html for this. It's already in use by Argo.

Reply:
I mean this should fix whatever is going on in argo with applied filters because they look off.

<img width="337" alt="Screenshot 2024-08-27 at 10 40 08 AM" src="https://github.com/user-attachments/assets/16e77b0f-5875-47b6-95c4-a1779511234a">

Reply:
Do you see the same here? https://sul-dlss.github.io/component-library/selected_item/

Reply:
Yes.

Reply:
<img width="445" alt="Screenshot 2024-08-27 at 9 52 26 AM" src="https://github.com/user-attachments/assets/fe60f65c-c277-4f5d-9703-174a9514bdab">
This is what I see in argo. What browser are you using?

Reply:
Firefox

Reply:
Also I could be wrong, but I think these should be two different things that don't conflict. One is for overriding base styles in blacklight. The other is for apps like sul-requests and pre-assembly that allow you to choose from a list and have pills.

Reply:
@dnoneill I'm also using Firefox.  What version are you on?  I'm on 129.0.2.  I'm pretty sure these are the same thing.  Maybe we should meet with Darcy about this?

Reply:
115.14.0

Reply:
Yes we should probably have a quick meeting/huddle on this.

Reply:
115.14.0 is quite old.   Can you upgrade?

Reply:
Probably, but I am using what is from self service so a lot of stanford is probably using it.

Reply:
Not according to google analytics.  Only 3.8% use Firefox.  And only fraction of those are using 115.

Reply:
About 0.5% according to https://analytics.google.com/analytics/web/#/analysis/p342534663/edit/nzbw4VEoRAuSHlUfi_kbvg